robots.txt

```diff
@@ -33,4 +33,6 @@ Disallow: /datacenter/ucp/1.1
 
 Disallow: /apidocs/v*
 
+Disallow: /machine/
```

Comment:
Wasn't there an option to set this in the metadata of the files that should not be indexed? (that way we wouldn't have to update the robots.txt)

Reply:
Not aware of the option to set the metadata of the file. Could you provide more details @thaJeztah?

Reply:
looks if we set `hide_from_sitemap` in the metadata; https://github.com/docker/docker.github.io/blob/24718c5e47eb85b7b219d34f13a4460df3616984/_layouts/docs.html#L105

Reply:
I guess it has to be set on a per-page level and not at the directory level? Any idea if affects both  docs.docker.com and Google search results? Thanks.

Reply:
Removed the update from robots.txt after discussing with @thaJeztah.